composer.json

```diff
@@ -9,7 +9,7 @@
         "ext-json": "*",
         "friendsofsymfony/elastica-bundle": "^5.1",
         "ruflin/elastica": "^6.1",
-        "sylius/sylius": "^1.6",
+        "sylius/sylius": "~1.7",
```

Comment:
remove php 7.2 from travis testing or
"^1.6|^1.7" => but then probably composer.lock in test application will be needed with either 1.6 and sonata bundles or with 1.7 and without sonata bundles